hexa/workspaces/models.py

```diff
@@ -55,6 +60,11 @@ def create_if_has_perm(
         if description is None:
             create_kwargs["description"] = "This is a workspace for {}".format(name)
 
+        db_name = format_db_name(create_kwargs["slug"])
+        create_kwargs["db_name"] = db_name
+        create_kwargs["db_password"] = get_db_server_credentials()["password"]
```

Comment:
What I understand here is that you set the password to be the one used by the user that creates the database. 

Each workspace has to have a unique db and a unique password.